<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToe</class>
 <widget class="QMainWindow" name="TicTacToe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TicTacToe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="posButton_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="posButton_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goSecondButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Second</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
